geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,591 @@
+      } else {
+        // when editor stopped, re-enable polygon & polyline, enable point/multipoint if vertex tool selected
+        pointButton.disableProperty().bind(geometryEditor.toolProperty().isEqualTo(freehandTool));
+        multipointButton.disableProperty().bind(geometryEditor.toolProperty().isEqualTo(freehandTool));
+        polylineButton.disableProperty().bind(new SimpleBooleanProperty(false));
+        polygonButton.disableProperty().bind(new SimpleBooleanProperty(false));
+      }
+    });
+
+    // bind the geometry editor tool to the tool chosen in the combo box
+    toolComboBox.valueProperty().bindBidirectional(geometryEditor.toolProperty());
+
+    // undo/redo enabled when the geometry editor can undo/redo
+    undoButton.disableProperty().bind(geometryEditor.canUndoProperty().not());
+    redoButton.disableProperty().bind(geometryEditor.canRedoProperty().not());
+
+    // delete selected element enabled only when an element is selected and deletable
+    ChangeListener<GeometryEditorElement> selectedElementChangedListener = (ob, ov, nv) -> {
+      if (nv == null) {
+        deleteSelectedElementButton.disableProperty().bind(new SimpleBooleanProperty(true));
+      } else {
+        deleteSelectedElementButton.disableProperty().bind(geometryEditor.getSelectedElement().canDeleteProperty().not());
+      }
+    };
+    geometryEditor.selectedElementProperty().addListener(selectedElementChangedListener);
+
+    // save/discard buttons enabled when geometry editor has started
+    stopAndSaveButton.disableProperty().bind(geometryEditor.startedProperty().not());
+    stopAndDiscardButton.disableProperty().bind(geometryEditor.startedProperty().not());
+
+    // delete all geometries button enabled when geometry editor stopped
+    deleteAllGeometriesButton.disableProperty().bind(geometryEditor.startedProperty());
+  }
+
+  /**
+   * Creates unique marker graphics for the different types of geometry.
+   */
+  private void createGraphicsMarkers() {
+    // orange-red square for points
+    pointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, Color.ORANGERED, 10);
+    // yellow circle for multipoints
+    multipointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, Color.YELLOW, 5);
+    // thin blue line for polylines
+    lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLUE, 2);
+    // black outline for polygons
+    SimpleLineSymbol polygonLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DASH, Color.BLACK, 1);
+    // translucent red interior for polygons
+    fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, Color.rgb(255, 0, 0, 0.3), polygonLineSymbol);
+  }
+
+  /**
+   * Starts the geometry editor with the point geometry type.
+   */
+  @FXML
+  private void handlePointButtonClicked() {
+    if (!geometryEditor.isStarted()) {
+      graphicsOverlay.clearSelection();
+      geometryEditor.start(GeometryType.POINT);
```

Comment:
```suggestion
      geometryEditor.setTool(vertexTool);
      geometryEditor.start(GeometryType.POINT);
```
We should set the vertex tool when we use the point geometry type, follows Qt.